environments/sandbox/sandbox-platform-hmcts-net.yml

```diff
@@ -125,6 +125,14 @@ A:
     record:
     - 10.50.11.10
     ttl: 300
+  - name: panorama-uks-0
+    record:
+    - 10.48.0.68
+    ttl: 300
+  - name: panorama-ukw-0
+    record:
+    - 10.100.198.164
+    ttl: 300
```

Comment:
couldn't you just add via terraform? or by enabling auto registration on the zone?

Reply:
hey @timja, the terraform reads the `yml` file, looking at the repo. Is there another way to add this?

Reply:
add them in your terraform code or enable auto registration on the vnet you are deploying this into